templates/web.template.yml

```diff
@@ -62,8 +62,8 @@ run:
           PRECOMPILE_ON_BOOT=1
         fi
         if [ -f /usr/local/bin/create_db ] && [ "$CREATE_DB_ON_BOOT" = "1" ]; then /usr/local/bin/create_db; fi;
```

Comment:
Should there be `|| exit 1` here, like the below? What happens if `create_db` fails?